serverless.yaml

```diff
@@ -140,6 +142,7 @@ resources:
         Description: "The deployment stage (e.g. dev, qa, prod). Default: dev"
       KibanaUserEmail:
         Type: String
+        Default: ${self:custom.kibanaUserEmail}
```

Comment:
I think this is okay but not setting a default on sls side. Also I think it is fine if we don't initialize the Kibana user automatically -- the kibana experience is very niche and only for developers

Reply:
Should we just remove all the Kibana User Pool user creation stuff then?

Reply:
We slacked, but yes we should!